Sources/ContainerClient/Utility.swift

```diff
@@ -165,11 +165,26 @@ public struct Utility {
         var config = ContainerConfiguration(id: id, image: description, process: pc)
         config.platform = requestedPlatform
 
+        let effectiveStorage: String? = {
+            if let storage = resource.storage {
```

Comment:
Perhaps we should separately validate the cases of storage provided through the command vs provided through the user default, and throw two different invalid argument errors so that the user can see what to correct in the case of the user default (in the unlikely case that they set the user default incorrectly via the `defaults` command or via Swift code).